### PR TITLE
SITES-11252: Limit formatting options for RTE editor (e.g., only bold…

### PR DESCRIPTION
**Pull request by irenelagno**
… allowed)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [ ] I have signed the [Adobe Open Source CLA](https://opensource.adobe.com/cla.html).
- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [ ] All new and existing tests passed.

### CODE REVIEW

File: cf-editor-rte-toolbar-configurable-buttons/src/aem-cf-editor-1/web-src/src/utils.js [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+  }
+  try {
+    content = JSON.parse(content)
+  } catch (e) {
+    // response is not json
+  }
+  return content
+}
+
+export const getContentFragmentConfig = async (token, imsOrg) => {
```
Comment (fe-lix-) [inline cf-editor-rte-toolbar-configurable-buttons/src/aem-cf-editor-1/web-src/src/utils.js:57]: This should be `getContentFragmentModelConfig`.

File: cf-editor-rte-toolbar-configurable-buttons/src/aem-cf-editor-1/actions/getCFModelConfig/index.js [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+    // }
+    // const content = await res.json()
+
+    //allowed buttons
+    const inputString = "aligncenter | alignjustify | alignleft | alignnone | alignright | blockquote | backcolor | bold | copy | cut | fontselect | fontsizeselect | forecolor | formatselect | h1 | h2 | h3 | h4 | h5 | h6 | indent | italic | language | lineheight | newdocument | outdent | paste | redo | remove | removeformat | selectall | strikethrough | styleselect | subscript | superscript | underline | undo | visualaid | pastetext | link | openlink | unlink | table | tablecellprops | tablecopyrow | tablecutrow | tabledelete | tabledeletecol | tabledeleterow | tableinsertdialog | tableinsertcolafter | tableinsertcolbefore | tableinsertrowafter | tableinsertrowbefore | tablemergecells | tablepasterowafter | tablepasterowbefore | tableprops | tablerowprops | tablesplitcells | tableclass | tablecellclass | tablecellvalign | tablecellborderwidth | tablecellborderstyle | tablecaption | tablecellbackgroundcolor | tablecellbordercolor | tablerowheader | tablecolheader | code | fullscreen | bullist | numlist | charmap | preview | searchreplace | visualblocks | insertdatetime | media | anchor";
```
Comment (fe-lix-) [inline cf-editor-rte-toolbar-configurable-buttons/src/aem-cf-editor-1/actions/getCFModelConfig/index.js:56]: Yes, that's exactly what I had in mind. To have a list in a parsable format. The only problem with your implementation is due to our current API. We can only remove and state what toolbar we actually want but this looks great.

I would assume the string coming from the CFM metadata would be JSON, but that's a minor detail.

**Review by fe-lix- — APPROVED** [review]
LGTM

File: cf-editor-rte-toolbar-configurable-buttons/README.md [hunk 3]
```diff
@@ -0,0 +1,49 @@
+# Configurable Buttons for Rich Text Editor
+
+Welcome to my Adobe I/O Application!
```
Comment (lenaorobei) [inline cf-editor-rte-toolbar-configurable-buttons/README.md:3]: Can you please add more details on how this extension works? Thanks!

File: cf-editor-rte-toolbar-configurable-buttons/src/dx-excshell-1/web-src/src/index.js [hunk 4]
```diff
@@ -0,0 +1,73 @@
+/*
```
Comment (lenaorobei) [inline cf-editor-rte-toolbar-configurable-buttons/src/dx-excshell-1/web-src/src/index.js:1]: This file can be as simple as

``` js
import "core-js/stable";

window.React = require("react");
import ReactDOM from "react-dom";

import App from "./components/App";

ReactDOM.render(<App />, document.getElementById("root"));
```

**Review by lenaorobei — CHANGES_REQUESTED** [review]
Please clean up unused code and add license headers to the files.

**Comment by irenelagno** [comment]
@lenaorobei fixed. Please check

**Review by lenaorobei — APPROVED** [review]

**Pull request merged**